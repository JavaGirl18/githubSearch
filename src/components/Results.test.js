import React from "react";
import { render } from "@testing-library/react";
import { Results } from ".";

test("it renders the results", () => {
  const props = [
    {
      id: 1,
      avatar_url: "https://avatars1.githubusercontent.com/u/57936?v=4",
      login: "test1",
      url: "https://api.github.com/users/example/1"
    },
    {
      id: 2,
      avatar_url: "https://avatars1.githubusercontent.com/u/57936?v=3",
      login: "test2",
      url: "https://api.github.com/users/example/2"
    }
  ];

  const { container } = render(<Results items={props} />);

  expect(container.firstChild).toMatchSnapshot();
});
