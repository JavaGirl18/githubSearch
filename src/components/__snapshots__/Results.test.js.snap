// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders the results 1`] = `
<div
  class="grid"
>
  <div
    class="card"
  >
    <img
      alt=""
      class="image"
      src="https://avatars1.githubusercontent.com/u/57936?v=4"
    />
    <div
      class="container"
    >
      <h4>
        <b>
           Owner: 
          test1
        </b>
      </h4>
      <p
        style="word-break: break-word;"
      >
        Repository URL:
         
        <a
          href="https://api.github.com/users/example/1"
          rel="noopener noreferrer"
          target="_blank"
        >
          https://api.github.com/users/example/1
        </a>
      </p>
      <p
        style="word-break: break-word;"
      >
        Forks: 
      </p>
    </div>
  </div>
  <div
    class="card"
  >
    <img
      alt=""
      class="image"
      src="https://avatars1.githubusercontent.com/u/57936?v=3"
    />
    <div
      class="container"
    >
      <h4>
        <b>
           Owner: 
          test2
        </b>
      </h4>
      <p
        style="word-break: break-word;"
      >
        Repository URL:
         
        <a
          href="https://api.github.com/users/example/2"
          rel="noopener noreferrer"
          target="_blank"
        >
          https://api.github.com/users/example/2
        </a>
      </p>
      <p
        style="word-break: break-word;"
      >
        Forks: 
      </p>
    </div>
  </div>
</div>
`;
